ts/screens/services/ServiceDetailsScreen.tsx

```diff
@@ -258,88 +188,63 @@ class ServiceDetailsScreen extends React.Component<Props, State> {
-              />
-
-              {(metadata.app_android ||
-                metadata.app_ios ||
-                metadata.web_url) && (
-                <H4 style={styles.infoHeader}>
-                  {I18n.t("services.otherAppsInfo")}
-                </H4>
-              )}
-              {metadata.web_url &&
-                renderInformationRow(
-                  I18n.t("services.otherAppWeb"),
-                  metadata.web_url,
-                  metadata.web_url
-                )}
-              {metadata.app_ios &&
-                renderInformationImageRow(
-                  I18n.t("services.otherAppIos"),
-                  metadata.app_ios,
-                  iOSStoreBadge
-                )}
-              {metadata.app_android &&
-                renderInformationImageRow(
-                  I18n.t("services.otherAppAndroid"),
-                  metadata.app_android,
-                  playStoreBadge
-                )}
-            </View>
-          )}
-        </React.Fragment>
+        <>
+          <Markdown
+            animated={true}
+            onLoadEnd={this.onMarkdownEnd}
+            onError={this.onMarkdownEnd}
+          >
+            {potServiceMetadata.value.description}
+          </Markdown>
+          <View spacer={true} large={true} />
+        </>
       );
     }
-    return undefined;
+    return null;
   };
 
-  private renderContactItems = (potServiceMetadata: ServiceMetadataState) => {
+  private renderServiceMetaData(potServiceMetadata: ServiceMetadataState) {
     if (pot.isSome(potServiceMetadata) && potServiceMetadata.value) {
       const metadata = potServiceMetadata.value;
+      const service = this.service;
+
       return (
-        <React.Fragment>
-          {renderRowWithDefinedValue(
-            metadata.address,
-            I18n.t("services.contactAddress"),
-            undefined,
-            "MAP"
-          )}
-          {renderRowWithDefinedValue(
-            metadata.support_url,
-            I18n.t("services.contactSupport")
-          )}
-          {renderRowWithDefinedValue(
-            metadata.phone,
-            I18n.t("services.contactPhone"),
-            "tel:"
-          )}
-          {renderRowWithDefinedValue(metadata.email, "Email", "mailto:")}
-          {renderRowWithDefinedValue(metadata.pec, "PEC", "mailto:")}
-          {renderRowWithDefinedValue(metadata.web_url, "Web")}
-        </React.Fragment>
+        <>
+          <TosAndPrivacyBox
+            tosUrl={metadata.tos_url}
+            privacyUrl={metadata.privacy_url}
+          />
+          <View spacer={true} large={true} />
+
+          <ContactPreferencesToggles
+            serviceId={service.service_id}
+            channels={service.available_notification_channels}
+          />
+          <View spacer={true} large={true} />
+
+          <ServiceMetadata
+            address={metadata.address}
+            androidStoreUrl={isAndroid ? metadata.app_android : undefined}
```

Comment:
```suggestion
            androidStoreUrl={isAndroid ? metadata.app_android : undefined}
```
Could it be an iOS user interested also to the Android app of the specific service, and viceversa? (maybe if he/she wants to share with a friend)
I have no strong opinions on this 

cc @thisisjp (this code shows the app store link only when there is a matching with the running OS. e.g: if the service has an Android app and the user is running IO on Android, then the link is shown, not otherwise)

Reply:
@Undermaken that's a good feature for a forked app called "NOI" :trollface: Let's only show the link related to the OS in use :)

Reply:
This was asked explicetly as a requirement.